apps/src/lab2/views/Lab2Wrapper.tsx

```diff
@@ -23,7 +23,11 @@ export interface Lab2WrapperProps {
 const Lab2Wrapper: React.FunctionComponent<Lab2WrapperProps> = ({children}) => {
   const isLoading: boolean = useSelector(isLabLoading);
   const isPageError: boolean = useSelector(hasPageError);
-
+  const errorMessage: string | undefined = useSelector(
+    (state: {lab: LabState}) =>
+      state?.lab?.pageError?.errorMessage ||
+      state?.lab?.pageError?.error?.message
```

Comment:
nit: don't think we need the optional for `state?.lab` as it should always be defined